### DISCUSSION TITLE
Strange characters show up for each line

### DISCUSSION THREAD

**Discussion by iongion** [sourcelair/xterm.js#5193] [Q&A]
1. I collect container logs using docker api that retrieves them as binary buffer
2. I have the current setup

```
      const terminal = new XTermTerminal({
        convertEol: true,
        drawBoldTextInBrightColors: true,
        rescaleOverlappingGlyphs: true,
        rightClickSelectsWord: true,
        fontSize: 12,
        fontFamily: `Consolas, "SF Mono", "DejaVu Sans Mono", "Droid Sans Mono", "Ubuntu Mono", "Roboto Mono", monospace`,
        disableStdin: true,
        scrollback: 16 * 1024,
        logLevel: "debug",
        allowProposedApi: true,
        fontWeight: "normal"
      });
```

What I see in the terminal and the text data itself looks like this

![image](https://github.com/user-attachments/assets/dacd0103-cb5a-4c6a-b091-e8b739aaff9a)

I don't understand those `K`, `x`, `q` in front of every log line date string, if anyone has any advice

**Comment by jerch** [comment]
Well your input contains spurious bytes, e.g. those [1,0,0,...] - they cannot be handled by a terminal, thus the warning. Why they in the data stream is unclear from what you provided, could it be a wrong offset at the byte scraping stage? Or maybe a message frame / protocol header, thats meant to be skipped for the data part?
